cribl_stream/README.md

```diff
@@ -0,0 +1,72 @@
+# Cribl Stream
+
+## Overview
+[Cribl Stream][1] helps you process machine data – logs, instrumentation data, application data, metrics, and so on – in real time, and deliver it to your analysis platform of choice. It allows you to:
```

Comment:
I just want to confirm that Cribl stream is the on-prem solution that we discussed?

Reply:
Correct, this will be for our self-hosted on-prem solution to start.